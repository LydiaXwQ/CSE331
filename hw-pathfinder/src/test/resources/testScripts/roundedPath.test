# Find a path with Dijkstra's algorithm when
# the path cost is different before and after rounding.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 3.28971
AddEdge simpleGraph B C 4.55432

FindPath simpleGraph A C