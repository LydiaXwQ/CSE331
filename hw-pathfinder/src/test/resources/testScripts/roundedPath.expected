# Find a path with Dijkstra's algorithm when
# the path cost is different before and after rounding.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 3.290 from A to B in simpleGraph
added edge 4.554 from B to C in simpleGraph

path from A to C:
A to B with weight 3.290
B to C with weight 4.554
total cost: 7.844